README.md

```diff
@@ -3,21 +3,39 @@
 [![CircleCI](https://circleci.com/gh/filecoin-project/go-fil-markets.svg?style=svg)](https://circleci.com/gh/filecoin-project/go-fil-markets)
 [![codecov](https://codecov.io/gh/filecoin-project/go-fil-markets/branch/master/graph/badge.svg)](https://codecov.io/gh/filecoin-project/go-fil-markets)
 
-This repository contains modular implementations of the storage and retrieval market subsystems of Filecoin. These modules are guided by the [v1.0 and 1.1 Filecoin specification updates](https://filecoin-project.github.io/specs/#intro__changelog). 
+This repository contains modular implementations of the [storage and retrieval market subsystems](https://filecoin-project.github.io/specs/#systems__filecoin_markets) of Filecoin. They are guided by the [v1.0 and 1.1 Filecoin specification updates](https://filecoin-project.github.io/specs/#intro__changelog). 
 
-Separating an implementation into a blockchain component and one or more mining and market components presents an opportunity to encourage implementation diversity while re-using non-security-critical components, and also greatly ease miner-operator customisations.
+Separating implementations into a blockchain component and one or more mining and market components presents an opportunity to encourage implementation diversity while reusing non-security-critical components.
 
 ## Components
 
-* [filestore](./filestore), ... 
+* **[storage market](./storagemarket)**: for finding, negotiating, and consummating deals to store data between clients and providers (storage miners)
+* **[retrieval market](./retrievalmarket)**: for finding, negotiating, and consummating deals to retrieve data between clients and providers (retrieval miners)
+* **[pieceio](./pieceio)**: utilities that take IPLD graphs and turn them into pieces
+* **[filestore](./filestore)**: a submodule branch that is a side effect of using the ffi to generate commP
+
+Related components in other repos:
+* **[data transfer](https://github.com/filecoin-project/go-data-transfer)**: for exchanging piece data between clients and miners, used by storage & retrieval market modules.
+
+### Background reading
+* The [Markets in Filecoin](https://filecoin-project.github.io/specs/#systems__filecoin_markets) section of the Filecoin Specification contains the canonical spec
+* The [Storage Market Module design doc](https://docs.google.com/document/d/1FfMUpW8vanR9FrXsybxBBbba7DzeyuCIN2uAXgE7J8U) is a more specific overview of these component implementations
+
+## Usage
+
+**Requires go 1.13**
+
+Install the module in your package or app with `go get "github.com/filecoin-project/go-fil-markets/filmarkets"`
```

Comment:
Install the go-fil-markets module that you want to use.  Available modules are:
* `filestore`
* `pieceio`
* `piecestore`
* `retrievalmarket`
* `storagemarket`

Install with:
`go get "github.com/filecoin-project/go-fil-markets/<module>"`